pythonforandroid/bootstraps/sdl2/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -71,6 +71,9 @@
                   android:configChanges="mcc|mnc|locale|touchscreen|keyboard|keyboardHidden|navigation|orientation|screenLayout|fontScale|uiMode{% if args.min_sdk_version >= 8 %}|uiMode{% endif %}{% if args.min_sdk_version >= 13 %}|screenSize|smallestScreenSize{% endif %}{% if args.min_sdk_version >= 17 %}|layoutDirection{% endif %}{% if args.min_sdk_version >= 24 %}|density{% endif %}"
                   android:screenOrientation="{{ args.manifest_orientation }}"
                   android:exported="true"
+                  {% if support_cutout %}
+                  android:theme="@style/BaseStyle"
+                  {% endif %}
```

Comment:
`android:theme` may be supported even if the device has no support for display-cutouts, and we will likely want to apply that feature to all the supported bootstraps.

Maybe, instead of `@style/BaseStyle`, we can rename it to `@style/KivyManagedStyle` ?